lib/game.rb

```diff
@@ -3,13 +3,51 @@
 class Game
   attr_reader :new_board
 
-  def start
-    @new_board = Board.new
-    puts "Welcome to Connect Four!"
-  end
-
   def initialize
     @new_board = nil
     start
   end
+
+  def start
+    @new_board = Board.new 
+  end
+
+  def print_board
+    7.times { |num| puts @new_board.board["row_#{num}".to_sym] }
+  end
+
+  def row_select
+    loop do
+      input = gets.chomp
+      if ["a", "b", "c", "d", "e", "f", "g"].include? input.downcase 
+        return input
+      else
+        puts "Invalid input. Please enter a letter from A - G"
+      end
+    end
+  end
+
+  def convert_input(input)
+    if input.downcase == "a"
```

Comment:
If we have time at the end we could potentially refactor this block using `case`

Reply:
True. Good call